feat(appconfig): add deploy method to configuration constructs (#28174)
aws/aws-cdk: packages/@aws-cdk/aws-appconfig-alpha/README.md

});
```

> If you want to deploy a configuration to an environment later and not on creation, you can call the `deploy` method on the configuration.

Reviewer: ```suggestion
To deploy a configuration to an environment after initialization use the `deployTo` method:
```
More concise.
